packages/react-app/src/components/giveaways/participation/ParticipationButton.tsx

```diff
@@ -175,7 +179,7 @@ const ParticipationButton = ({
       {showWinnerModal && (
         <WinnerModal
           giveaway={giveaway}
-          open={showWinnerModal}
+          open={true}
```

Comment:
If this is always true then the WinnerModal doesn't need the `open` prop. Whether a child component is shown or not should always be controlled by the parent component